mmv1/api/async.rb

```diff
@@ -66,6 +66,9 @@ class OpAsync < Async
     attr_reader :result
     attr_reader :status
     attr_reader :error
+    # If true, include project as an argument to OperationWaitTime. 
```

Comment:
```suggestion
    # If true, include project as an argument to OperationWaitTime.
```
Our linter doesn't like trailing whitespace in ruby